pkg/restapi/types.go

```diff
@@ -11,13 +11,6 @@ const (
 	SessionCanceling = "canceling"
 )
 
-const (
```

Comment:
can we separate these restapi changes out? this will break the desktop app

Reply:
Iwill update the desktop app